packages/bugsnag_flutter/lib/src/model/breadcrumbs.dart

```diff
@@ -9,11 +9,10 @@ class BugsnagBreadcrumb {
 
   final DateTime timestamp;
 
-  BugsnagBreadcrumb(
-    this.message, {
-    this.type = BugsnagBreadcrumbType.manual,
-    this.metadata,
-  }) : timestamp = DateTime.now().toUtc();
+  BugsnagBreadcrumb(this.message,
+      {this.type = BugsnagBreadcrumbType.manual, Map<String, Object>? metadata})
+      : timestamp = DateTime.now().toUtc(),
+        metadata = BugsnagMetadata.sanitizedMap(metadata ?? {});
```

Comment:
We should probably be sanitizing the data in `toJson()`. This avoids unexpectedly changing the visible content of `metadata` (a public API). It would also move the performance impact later in the breadcrumb lifecycle (ie: we only sanitize the data if it is converted to JSON), which is advantageous since ultimately most breadcrumbs are discarded.